src/autoschedulers/adams2019/CMakeLists.txt

```diff
@@ -2,6 +2,9 @@
 # Resources for the autoscheduler library
 ##
 
+set(COMMON_DIR "${Halide_SOURCE_DIR}/src/autoschedulers/common")
+include_directories("${Halide_SOURCE_DIR}/src/autoschedulers/adams2019")
```

Comment:
We do not use `include_directories` in this codebase. Please read `README_cmake.md`.

Reply:
I doubt this is necessary to begin with.